### ISSUE TITLE
Error: update failed for AnAction with ID=ExternalJavaDoc

### ISSUE THREAD

**Issue by fruitl00p** [psliwa/idea-composer-plugin#8]
Had this come up 2 times. I just switched projects.. ?

Hope the stackstrace is of any use!

```
update failed for AnAction with ID=ExternalJavaDoc
java.lang.NullPointerException
	at org.psliwa.idea.composerJson.intellij.codeAssist.schema.SchemaDocumentationProvider.loop$2(SchemaDocumentationProvider.scala:69)
	at org.psliwa.idea.composerJson.intellij.codeAssist.schema.SchemaDocumentationProvider.findTokens(SchemaDocumentationProvider.scala:73)
	at org.psliwa.idea.composerJson.intellij.codeAssist.schema.SchemaDocumentationProvider.getUrlFor(SchemaDocumentationProvider.scala:24)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.hasUrlsFor(CompositeDocumentationProvider.java:222)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.hasDocumentationFor(CompositeDocumentationProvider.java:193)
	at com.intellij.ide.actions.ExternalJavaDocAction.update(ExternalJavaDocAction.java:126)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:113)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:250)
	at com.intellij.openapi.actionSystem.impl.Utils.b(Utils.java:209)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:147)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.g(ActionMenu.java:281)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$400(ActionMenu.java:41)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:235)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1043)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1122)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at com.jarego.jayatana.swing.SwingGlobalMenuWindow$3.run(SwingGlobalMenuWindow.java:346)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:302)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:745)
	at java.awt.EventQueue.access$300(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:706)
	at java.awt.EventQueue$3.run(EventQueue.java:704)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:715)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)



java.lang.NullPointerException
	at org.psliwa.idea.composerJson.intellij.codeAssist.schema.SchemaDocumentationProvider.loop$2(SchemaDocumentationProvider.scala:69)
	at org.psliwa.idea.composerJson.intellij.codeAssist.schema.SchemaDocumentationProvider.findTokens(SchemaDocumentationProvider.scala:73)
	at org.psliwa.idea.composerJson.intellij.codeAssist.schema.SchemaDocumentationProvider.getUrlFor(SchemaDocumentationProvider.scala:24)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.hasUrlsFor(CompositeDocumentationProvider.java:222)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.hasDocumentationFor(CompositeDocumentationProvider.java:193)
	at com.intellij.ide.actions.ExternalJavaDocAction.update(ExternalJavaDocAction.java:126)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:113)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:250)
	at com.intellij.openapi.actionSystem.impl.Utils.b(Utils.java:209)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:147)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.g(ActionMenu.java:281)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$400(ActionMenu.java:41)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:235)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1043)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1122)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at com.jarego.jayatana.swing.SwingGlobalMenuWindow$3.run(SwingGlobalMenuWindow.java:346)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:302)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:745)
	at java.awt.EventQueue.access$300(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:706)
	at java.awt.EventQueue$3.run(EventQueue.java:704)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:715)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
```

**Comment by psliwa** [comment]
What version of this plugin do you have installed? I think this is the same issue as #7 - it was fixed in 1.0.6 release.

**Comment by fruitl00p** [comment]
Indeed, i was still on 1.0.5, just updated, will re-open if it returns... !
